src/main/java/com/retrip/trip/domain/entity/Trip.java

```diff
@@ -0,0 +1,47 @@
+package com.retrip.trip.domain.entity;
+
+import com.retrip.trip.domain.vo.TripPeriod;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+import static lombok.AccessLevel.PROTECTED;
+
+@Getter
+@NoArgsConstructor(access = PROTECTED)
+@Entity
+public class Trip extends BaseEntity {
+    @Id
+    @Column(columnDefinition = "varbinary(16)")
+    private UUID id;
+    private UUID leaderId;
+
+    @Version
+    private long version;
+    private String title;
```

Comment:
title VO 방식으로 하는 거 제안 드립니다!
이런식으로 하면 @Valid 사용하는 것 보다 유효성 검사도 더 세부적으로 가능하고 테스트 할 때 더 간편하게 가능합니다!
```suggestion
    @Embedded
    private TripTitle title;

 @Getter
@Builder
@Embeddable
@EqualsAndHashCode
@NoArgsConstructor(access = AccessLevel.PROTECTED, force = true)
public class TripTitle {

    private static final int LENGTH_LIMIT = 30;

    @Column(name = "title", nullable = false, length = LENGTH_LIMIT)
    private final String value;

    public ListTitle(String value) {
        validate(value);
        this.value = value;
    }

    private void validate(String value) {
        if (value.length() > LENGTH_LIMIT) {
            throw new CustomException(LENGTH_EXCEEDED_EXCEPTION, "여행 제목은 " + LENGTH_LIMIT + "자를 넘을 수 없습니다.");
        }
    }
```

Reply:
VO 추가했습니다.